3 reports.sql

```diff
@@ -0,0 +1,49 @@
+/*1) I offer deep cleans for each customer on request. Deep cleans take twice as long as normal cleans.
+   I charge 50% more than a regular clean in that house. 
+   For this I need to see the customers phone number, last name and the amount of money they will have to pay.
+   Please do it in this format:  last name, first name (phone number) - price of service.
+*/
+SELECT DeepCleanCustomers= CONCAT(c.LastName, ' ', c.FirstName, ' (',c.PhoneNumber, ') - ', (c.PricePerHour*c.TimeinHours)  )
```

Comment:
Add a comma between last and first names.
Deep clean takes twice as long, and there's an extra charge of 50% more. You are only returning the twice as long calculation.